spec/acceptance/users_controller_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:resource) { Resource.first }
   let(:resource2) { Resource.second }
 
-  patch "users/me" do
+  patch "users/update" do
```

Comment:
this acceptance test can still run against `users/me`, it should automatically resolve to the current user